code/modules/vehicles/scooter.dm

```diff
@@ -0,0 +1,87 @@
+/obj/vehicle/scooter
+	name = "scooter"
+	desc = "A popular child's toy back on the planets, handcrafted by some greyshirt."
+	icon_state = "scooter"
+	vehicle_move_delay = 2.2 //slightly slower than other vehicles, due to being made with pipes, rods, and metal
+	var/pipetype = 1 //so the pipe created on deconstruction doesn't have a blank icon_state, changes on construction of the scooter
+	var/pipename	 //same
+/obj/vehicle/scooter/attackby(obj/item/I, mob/user, params)
+	if(istype(I, /obj/item/weapon/wrench))
+		user << "<span class='notice'>You begin to remove the pipe..</span>"
+		if(do_after(user, 40/I.toolspeed, target = src))
```

Comment:
you should check and set a deconstructed flag, or check that src is still alive and not qdeleted, otherwise multiple people can start a deconstruction step on this and get multiple parts from a single scooter.